charts/cray-velero/Chart.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v2
+name: cray-velero
+version: 0.1.0
+description: Velero is an open source tool to safely backup and restore, perform disaster recovery, and migrate Kubernetes cluster resources and persistent volumes
+keywords:
+  - velero
+home: https://github.com/Cray-HPE/cray-velero
+maintainers:
+  - name: bklei
+  - name: cdelatte-hpe
+icon: https://cdn-images-1.medium.com/max/1600/1*-9mb3AKnKdcL_QD3CMnthQ.png
+sources:
+- https://github.com/vmware-tanzu/velero
+- https://github.com/vmware-tanzu/helm-charts
+appVersion: 2.14.1
+annotations:
+  artifacthub.io/changes: |
+    - kind: security
+      description: Refactor to use patched CSM images
+      links:
+        - name: Github Issue
+          url: https://github.com/Cray-HPE/cray-velero/issues/3
+        - name: Github PR
+          url: https://github.com/Cray-HPE/cray-velero/pull/2
+  artifacthub.io/images: |
+    - name: docker-kubectl
+      image: artifactory.algol60.net/csm-docker/stable/docker-kubectl:1.19.15
+    - name: velero
+      image: artifactory.algol60.net/csm-docker/stable/docker.io/velero/velero:v1.5.2
+    - name: velero-plugin-for-aws
+      image: artifactory.algol60.net/csm-docker/stable/docker.io/velero/velero-plugin-for-aws:v1.1.0
+    - name: velero-restic-restore-helper
+      image: artifactory.algol60.net/csm-docker/stable/docker.io/velero/velero-restic-restore-helper:v1.5.2
+  artifacthub.io/license: MIT
```

Comment:
The Chart.yaml doesn't have the requirements, so how does that work?

Reply:
🤷 Maybe because I ran `make dep-up` locally? Either way, deps need to be added to Chart.yaml. Fixed.